stream-tests/src/test/java/org/apache/pekko/stream/javadsl/BufferUntilChangedTest.java

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
this is the wrong header unless you have copied this code from somewhere else - the right header for 100% new code is https://github.com/apache/pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16